engine/lib/output.php

```diff
@@ -142,6 +142,11 @@ function elgg_format_attributes(array $attrs = array()) {
 	$attributes = [];
 
 	foreach ($attrs as $attr => $val) {
+		if (!is_string($attr) || $attr[0] == '#') {
```

Comment:
the leaking of hashed attributes was fixed by me in https://github.com/Elgg/Elgg/pull/10421/commits/fd9958c0f5ea824e99fc1881a3c416fb3342cdcb